modules/ztp-syncing-new-configmap-changes-to-existing-pgt-crs.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-syncing-new-configmap-changes-to-existing-pgt-crs_{context}"]
+= Syncing new ConfigMap changes to existing PolicyGenTemplate CRs
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a `PolicyGenTemplate` CR that pulls information from a `ConfigMap` CR using hub cluster templates.
+
+.Procedure
+
+. Update the contents of your `ConfigMap` CR, and apply the changes in the hub cluster.
+
+. To sync the contents of the updated `ConfigMap` CR to the deployed policy, do either of the following:
+
+.. Optional: Delete the existing policy and recreate it using the `PolicyGenTemplate` CR.
+
+... Delete the policy, for example:
++
+[source,terminal]
+----
+$ oc delete policy <policy_name> -n <policy_namespace>
+----
++
+... Re-apply the `PolicyGenTemplate` CR:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
+----
+
+.. Optional: Apply a special annotation `policy.open-cluster-management.io/trigger-update` to the policy with a different value every time when updating the `ConfigMap`. For example, run the following command:
++
+[source,terminal]
+----
+$ $ oc label <policy_name> -n <policy_namespace> policy.open-cluster-management.io/trigger-update="1"
+----
+
+... Re-apply the `PolicyGenTemplate` CR to update the applied policy. For example, run the following command:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
```

Comment:
This should be removed as the policy has already been updated by the above command 
```suggestion
```